.env.example

```diff
@@ -17,6 +17,7 @@ LOGBOOK_BASE_URL="http://localhost:3030/scichatapi"
 METADATA_KEYS_RETURN_LIMIT=100
 METADATA_PARENT_INSTANCES_RETURN_LIMIT=100
 MONGODB_URI="mongodb://<USERNAME>:<PASSWORD>@<HOST>:27017/<DB_NAME>"
+MONGDB_TEST_NAME="scicat-test"
```

Comment:
Why do we need this variable? The database name is already included in MONGODB_URI.
IF we are in a testing environment, we should change MONGODB_URI locally.
If we are going to keep it, can you fix the misspelling? It should MONGODB_TEST_NAME